### DISCUSSION TITLE
Docker daemon might not be running error

### DISCUSSION THREAD

**Discussion by sahithyandev** [supabase/supabase#12964] [Questions]
While I was trying to get started with Supabase, I got a somewhat verbose error that I couldn't figure out what was wrong. I followed Supabase Docs without anything new.
Maybe someone can help me figure out what I am doing wrong?

Reproduction here: https://github.com/sahithyandev/supabase-issue

**Comment by gaslitbytech** [comment]
This error is the same as this: https://github.com/docker/for-win/issues/13137

**Reply by sahithyandev** [reply]
But... I haven't installed Docker in my system. How can I fix the issue?

**Reply by gaslitbytech** [reply]
> I haven't installed Docker in my system.

This is the issue. You must install docker.

**Reply by sahithyandev** [reply]
I see. Thanks.
